pkg/daemon/daemon.go

```diff
@@ -201,6 +202,10 @@ func (d *Daemon) WaitUntilState(expected types.DaemonState) error {
 	_, err, shared := d.stateGetterGroup.Do(d.ID(), stateGetter)
 	log.L.Debugf("Get daemon %s with shared result: %v ", d.ID(), shared)
 
+	if exportErr := exporter.ExportNydusdEventMetric(d.States.ID, string(expected)); exportErr != nil {
```

Comment:
We should check `err` if it is nil. Then record the metric

Reply:
Fixed.